debian/rules

```diff
@@ -5,18 +5,9 @@ PACKAGE = $(shell dh_listpackages|head -n 1|cut -d '-' -f 1)
 SYSCONFDIR = "/etc"
 DESTDIR = "debian/tmp"
 
-ifeq ($(shell lsb_release -sr), 14.04)
-        SUBSTVARS = -Vjre:Depends="openjdk-7-jre-headless" -Vjdk:Depends="openjdk-7-jdk" -Vinit:"Depends:jsvc"
-else
-        SUBSTVARS = -Vjre:Depends="openjdk-8-jre-headless" -Vjdk:Depends="openjdk-8-jdk" -Vinit:"Depends:systemd"
-endif
-
```

Comment:
@vincentbernat that's right, but if we use that then we'll need to build/host/publish two version of the deb repository, depending on whether the deb pkgs were built of Ubuntu 14.04 or 16.04+. I think the current is still a good compromise, I can add new dependency options to allow users to use oracle jre/jdk.